apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/StorageActivity.java

```diff
@@ -4,12 +4,8 @@
 import android.support.annotation.Nullable;
 import android.support.v7.app.AppCompatActivity;
 import android.util.Log;
-
-import com.google.gson.annotations.Expose;
 import com.microsoft.appcenter.sasquatch.R;
```

Comment:
blank line before this import

Reply:
hmmm must be weird commit history in github, I did not see that in my local file. Meanwhile I am running code formatting over the whole module, should fix this...